src/test/java/ibm/jceplus/junit/base/memstress/BaseTestMemStressXDH.java

```diff
@@ -17,24 +18,18 @@
 import java.security.spec.NamedParameterSpec;
 import java.util.Arrays;
 import javax.crypto.KeyAgreement;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.Assert.assertTrue;
 
-public class BaseTestMemStressXDH extends ibm.jceplus.junit.base.BaseTest {
+public class BaseTestMemStressXDH extends BaseTestJunit5 {
     /* This class by default tests "X25519" */
 
     int numTimes = 100;
     boolean printheapstats = false;
     String curveName = "X25519";
 
-    //"X448";
-    public BaseTestMemStressXDH(String providerName) {
-        super(providerName);
-    }
-
-    public BaseTestMemStressXDH(String providerName, String curveName) {
-        super(providerName);
-        this.curveName = curveName;
```

Comment:
Do we have a way to set to a different curve?

Reply:
Yes we can use getAlgorithm(). I made this update in the latest code.